pkg/agent/route/interfaces.go

```diff
@@ -51,6 +52,24 @@ type Interface interface {
 	// DeleteSNATRule should delete rule to SNAT outgoing traffic with the mark.
 	DeleteSNATRule(mark uint32) error
 
+	// InitService should add the basic TC configuration on Linux.
```

Comment:
I know existing funcs go this way, but I suggest to change "should add" to "adds". The same for other funcs you added.

Reply:
Thanks for reminding.